### ISSUE TITLE
Future request

### ISSUE THREAD

**Issue by DearZack** [PasteBar/PasteBarApp#167]
if I "pin" or "start" one，Can we achieve automatic deletion when pressing 'Clear All History'

**Comment by DearZack** [comment]
Is there a plan to do a 'one click backup' when I change my computer. Including data and user configuration

**Comment by kurdin** [comment]
> if I "pin" or "start" one，Can we achieve automatic deletion when pressing 'Clear All History'
@DearZack Can you please explain it again, not sure if I understand what functionality you want. 

> Is there a plan to do a 'one click backup' when I change my computer. Including data and user configuration
Yes, everything is stored as a single file, just look for `pastebar-db.data` filename on your system. You can move this file to a new computer or copy it for a backup.

**Comment by DearZack** [comment]
![image](https://github.com/user-attachments/assets/68edbc0e-307f-4875-a971-e16ed9083b9c)
![image](https://github.com/user-attachments/assets/385d695c-4694-409c-9512-9aed62877a0f)

if I pinned，when i “clear all history”，  i want the pinned item will not be  removed

**Comment by DearZack** [comment]
If there is currently no plan for the above situation, can a "Paste Menu window" be added to the "Global System OS Hotkeys" for the "Paste Menu"?

**Comment by DearZack** [comment]
My usage scenario is that I will input some text in advance for easy and quick input. Although "Pin" will also make it easier for me to input quickly in the future, I like to timed "Clear All History", which may result in the loss of data from my previous "Pin"

**Comment by kurdin** [comment]
@DearZack Yeah, I understand what you mean/want. I don't think `Clear All History` should keep pinned / starred history items. It would be strange to have. I think of history starred/pinned items as temporary solution for quick tag and access recently copied items. If you want something permanent, just save your important items as saved clips. You can star/pin saved clips as well. 

I like your idea to have Global System OS Hotkeys for `Paste Menu window` similar to `Quick Paste window` so we can access and paste clips from Paste Menu quickly. I'll add this functionality in future releases. 

Thank you @DearZack for your ideas !
